### PR TITLE
codeql-analysis

### LINKED ISSUE #34: GHA Vulnerability Scan
There should be a GHA job that scans the application for vulnerability on web and Backend

### PR DESCRIPTION
**Pull request by nextwebb**
Closes #34 . Vulnerability Scan workflow using Using codeql-analysis to scan your Git repo

### CODE REVIEW

**Review by adefemi171 — APPROVED** [review]
I think I will go with this vulnerability scanning tool rather than #93 because the action shows all the files that is been extracted and scanned, also try to resolve the mergeable issue you are having. Other than that this LGTM.

**Comment by nextwebb** [comment]
> I think I will go with this vulnerability scanning tool rather than #93 because the action shows all the files that is been extracted and scanned, also try to resolve the mergeable issue you are having. Other than that this LGTM.
I'll add some description.

**Comment by nextwebb** [comment]
@adefemi171 made some updates.

**Review by adefemi171 — DISMISSED** [review]

**Comment by nextwebb** [comment]
@dabsclement @phawazzzy @AminuIsrael @Robogeek95  this PR needs review

File: .github/workflows/codeql-analysis.yml [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ master ]
+  schedule:
+    - cron: '21 21 * * 2'
```
Comment (phawazzzy) [inline .github/workflows/codeql-analysis.yml:21]: This means everyday  by 2am right

Comment (nextwebb) [reply .github/workflows/codeql-analysis.yml:21]: it means “At 21:21 on Tuesday.”
but its important. lemme remove it.

**Review by nextwebb — COMMENTED** [review]
@adefemi171 @phawazzzy fixed!

**Comment by phawazzzy** [comment]
> @adefemi171 @phawazzzy fixed!

so now it is on every PR right

**Review by phawazzzy — APPROVED** [review]

**Comment by nextwebb** [comment]
> > @adefemi171 @phawazzzy fixed!
> 
> so now it is on every PR right
yes

**Comment by nextwebb** [comment]
@AminuIsrael @dabsclement @Robogeek95 can you help with reviewing this. thanks

**Review by AminuIsrael — APPROVED** [review]

**Pull request merged**